pkg/inframetadata/internal/hostmap/hostmap.go

```diff
@@ -0,0 +1,89 @@
+package hostmap
+
+import (
+	"fmt"
+	"sync"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.uber.org/multierr"
+
+	"github.com/DataDog/opentelemetry-mapping-go/pkg/inframetadata/gohai"
+	"github.com/DataDog/opentelemetry-mapping-go/pkg/inframetadata/payload"
+)
+
+// HostMap stores a map from hostnames to host metadata payloads.
```

Comment:
```suggestion
// HostMap maps hostnames to host metadata payloads.
```
The rest seems like documentation for its methods. It is quite a simple API, so viewing this within the context of "godoc" will very easily reveal all that it's about. It only has 2 methods.

Reply:
Removed the extra documentation; applied the suggested change